src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/resource/ResyncAlertsIntTest.kt

```diff
@@ -162,6 +164,51 @@ class ResyncAlertsIntTest : IntegrationTestBase() {
     assertThat(deletedAlert).isNull()
   }
 
+  @ParameterizedTest(name = "{0} allowed")
+  @ValueSource(strings = [ROLE_ALERTS_ADMIN, ROLE_NOMIS_ALERTS])
```

Comment:
This will run the test twice and the roles allowed for this are covered elsewhere?